logger/logger.go

```diff
@@ -37,6 +37,9 @@ type Logger interface {
 	Notice(msg string)
 	// Debug is for messages that the user should be able to find if they're debugging something
 	Debug(msg string)
+	// DebugAlways is for messages that we always want to print so the gating must
+	// be done by the caller
+	DebugAlways(msg string)
```

Comment:
Wondering about this name, mostly also because "DebugAlwaysf" is strange,  I don't have strong opinions but other options could be:

NoGuardDebug
DebugNotice
DebugLog